### PR TITLE
Run autopep8 over the whole tree

### PR DESCRIPTION
**Pull request by genodeftest**
This is the first step of #371.

### CODE REVIEW

File: xlgui/__init__.py [hunk 1]
```diff
@@ -252,8 +252,10 @@ def collection_manager(self, *e):
             new_libraries = sorted(dialog.get_items())
 
             if collection_libraries != new_libraries:
-                collection_locations = [location
-                                        for location, monitored, startup_scan in collection_libraries]
+                collection_locations = [
+                    location for location,
```
Comment (javiercantero) [inline xlgui/__init__.py]: Here the code readability of the line is worse with the new formatting.

**Comment by genodeftest** [comment]
> About 61630cd, I would have split the patchst in two: one about logic optimization (not in, explicit True/False removing) and another about long line formatting style.

I've split them now, into more than 2 different commits. I am excluding the "line width" related fixers for now, because they need more manual work. Since there are only a few PRs open, now is the chance to do this.

**Comment by javiercantero** [comment]
I don't know what is happening, but autopep8 didn't get all the cases. For example, look at xl/collection.py line 210 and line 689, the `not ... in` weren't rewritten to `not in`. However, the one in 686 did (by aaff2a2f0b1c5854de15253794868fed9f0059f6)

**Comment by javiercantero** [comment]
More info: running `pycodestyle --statistics -qq ./` returns these errors/warnings:

```
1       E114 indentation is not a multiple of four (comment)
1       E125 continuation line with same indent as next logical line
1       E201 whitespace after '('
1       E202 whitespace before ')'
112     E203 whitespace before ':'
22      E221 multiple spaces before operator
31      E231 missing whitespace after ','
18      E261 at least two spaces before inline comment
1       E262 inline comment should start with '# '
102     E265 block comment should start with '# '
44      E266 too many leading '#' for block comment
1       E271 multiple spaces after keyword
1       E272 multiple spaces before keyword
114     E305 expected 2 blank lines after class or function definition, found 1
108     E402 module level import not at top of file
1741    E501 line too long (81 > 79 characters)
15      E711 comparison to None should be 'if cond is not None:'
18      E712 comparison to True should be 'if cond is True:' or 'if cond:'
7       E713 test for membership should be 'not in'
10      E731 do not assign a lambda expression, use a def
8       E741 ambiguous variable name 'l'
117     W291 trailing whitespace
17      W293 blank line contains whitespace
```

A lot less than previously, but some of them should have completely gone.

**Comment by genodeftest** [comment]
> I don't know what is happening, but autopep8 didn't get all the cases. For example, look at xl/collection.py line 210 and line 689, the not ... in weren't rewritten to not in. However, the one in 686 did (by aaff2a2).

Thanks for the catch! I kinda expected this, from reading the `autopep8` documentation, especially if you have a look at the `--aggressive` parameter. This PR was just for getting somewhat like 90% to pep8-compliant code, not 100% – it was just about the low-hanging fruits. Manual fixes are required for the rest, but a few thousand fixes have been done automatically, which simplifies the work.

**Comment by genodeftest** [comment]
> More info: running pycodestyle --statistics -qq ./ returns these errors/warnings:
[…]
112     E203 whitespace before ':'

If you have a look at the code, at some places spaces were used to make the code look nicer, e.g. in https://github.com/exaile/exaile/blob/master/xl/metadata/mp4.py#L32. I did not want to include potentially controversial changes the first run, because the longer we discuss, the longer other work is blocked by this pull request.

Some of the other warnings are about potentially controversial changes too.

**Comment by virtuald** [comment]
My opinion is let's get a style checker running automatically (and maybe a git hook too? gstreamer does this, and it's pretty useful), then we can incrementally suppress the warnings that are stupid. I'm sure there will be a lot of them.

**Comment by genodeftest** [comment]
> My opinion is let's get a style checker running automatically (and maybe a git hook too? gstreamer does this, and it's pretty useful), then we can incrementally suppress the warnings that are stupid. I'm sure there will be a lot of them.

+1. See #371.

**Pull request merged**